packages/angular-cli/models/webpack-build-common.ts

```diff
@@ -33,7 +33,8 @@ export function getWebpackCommonConfig(
   return {
     devtool: 'source-map',
     resolve: {
-      extensions: ['.ts', '.js']
+      extensions: ['.ts', '.js'],
+      modules: [ path.join(projectRoot, 'node_modules') ]
```

Comment:
Could you use `modules: ['node_modules', path.resolve(projectRoot, 'node_modules')]` instead to have a fallback?